$OpenBSD: patch-testsuite_mk_test_mk,v 1.8 2018/01/22 00:42:30 kili Exp $

Disable LLVM tests for now (until I unslack and debug why all LLVM
tests are failing).

Index: testsuite/mk/test.mk
--- testsuite/mk/test.mk.orig
+++ testsuite/mk/test.mk
@@ -154,16 +154,7 @@ else
 RUNTEST_OPTS += -e ghc_with_smp=0
 endif
 
-ifeq "$(LLC)" ""
 RUNTEST_OPTS += -e ghc_with_llvm=0
-else ifeq "$(TargetARCH_CPP)" "powerpc"
-RUNTEST_OPTS += -e ghc_with_llvm=0
-else ifneq "$(LLC)" "llc"
-# If we have a real detected value for LLVM, then it really ought to work
-RUNTEST_OPTS += -e ghc_with_llvm=1
-else
-RUNTEST_OPTS += -e ghc_with_llvm=0
-endif
 
 ifeq "$(WINDOWS)" "YES"
 RUNTEST_OPTS += -e windows=True
